x/claims/keeper/migrations.go

```diff
@@ -0,0 +1,27 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+
+	v2 "github.com/tharsis/evmos/x/claims/migrations/v2"
+)
+
+var _ module.MigrationHandler = Migrator{}.Migrate1to2
+
+// Migrator is a struct for handling in-place store migrations.
+type Migrator struct {
+	keeper Keeper
+}
+
+// NewMigrator returns a new Migrator.
+func NewMigrator(keeper Keeper) Migrator {
+	return Migrator{
+		keeper: keeper,
+	}
+}
+
+// Migrate1to2 migrates from consensus version 1 to 2.
+func (m Migrator) Migrate1to2(ctx sdk.Context) error {
```

Comment:
suggest putting this migration in #352 instead, so that you can set the new params as well, and then you only need to do 1 migration, not two. You can't do the full migration in this PR for the `v1.1.2` release.

Reply:
sounds good, just removed it